draft-kazuho-httpbis-priority.md

```diff
@@ -345,6 +331,33 @@ order the requests were generated.  Doing so maximizes the chance of the client
 making progress in using the composition of the HTTP responses at the earliest
 moment.
 
+
+# The Priority HTTP Header Field
+
+The Priority HTTP header field can appear in requests and responses. A client
+uses it to specify the priority of the response. A server uses it to inform
+the client that the priority was overwritten. An intermediary can use the
+Priority information from client requests and server responses to correct or
+amend the precedence to suit it (see {{merging}}).
+
+Unknown parameters MUST be ignored.
```

Comment:
I think this belongs to the section above.

Reply:
Thanks, removed.